src/Compilers/CSharp/Portable/CommandLine/CSharpCommandLineParser.cs

```diff
@@ -552,7 +553,12 @@ internal sealed override CommandLineArguments CommonParse(IEnumerable<string> ar
                             continue;
 
                         case "nosdkpath":
-                            sdkDirectory = null;
+                            noSdkPath = true;
```

Comment:
Generally when conflicting arguments are passed to the compiler the last one wins (like `/checked`). Why are we handling `sdkpath` differently?

Reply:
Fixed, thanks.